pkg/supportbundle/helm.go

```diff
@@ -0,0 +1,154 @@
+package supportbundle
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	imagedocker "github.com/containers/image/v5/docker"
+	dockerref "github.com/containers/image/v5/docker/reference"
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kots/kotskinds/client/kotsclientset/scheme"
+	"github.com/replicatedhq/kots/pkg/docker/registry"
+	"github.com/replicatedhq/kots/pkg/logger"
+	"github.com/replicatedhq/kots/pkg/store"
+	"github.com/replicatedhq/kots/pkg/supportbundle/types"
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	troubleshootclientsetscheme "github.com/replicatedhq/troubleshoot/pkg/client/troubleshootclientset/scheme"
+	"github.com/replicatedhq/troubleshoot/pkg/docrewrite"
+	troubleshootsb "github.com/replicatedhq/troubleshoot/pkg/supportbundle"
+	"github.com/segmentio/ksuid"
+	"helm.sh/helm/v3/pkg/helmpath"
+	helmregistry "helm.sh/helm/v3/pkg/registry"
+)
+
+// TODO: support customer redactors
```

Comment:
there is already a function that returns this list called `redact.GetRedactInfo()`

Reply:
I'm making some SC stories for missing functionality here.